spec/controllers/items_controller_spec.rb

```diff
@@ -44,65 +44,114 @@
     end
 
     it "response redirect" do
-      post :create, :item => {name: "a", description: "A", price: 1, user_id: user.id}
+      post :create, :item => {name: "a", description: "A", buy_it_now_price: 1, user_id: user.id}
```

Comment:
When you added a new param required by item you had to update params in all the specs. Would it be possible to have the params defined in one place for this context?